.circleci/config.yml

```diff
@@ -44,19 +44,19 @@ jobs:
       - checkout
       - runtests
   
-  node-carbon: # EOL 2019-12-31
+  node-erbium: # EOL 2022-04-30
```

Comment:
Can we move away from these weird circleci tags and just use the node version (10, 12, 14, current?) for the tag? i.e. `node-12` and `circleci/node:12`.

Reply:
Great idea. Tried it, and it seems to have worked!